dist/backbone.collectionView.js

```diff
@@ -35,8 +35,8 @@
 		tagName : "ul",
 
 		events : {
-			"mousedown li, td" : "_listItem_onMousedown",
-			"dblclick li, td" : "_listItem_onDoubleClick",
+			"mousedown > li, > tr > td" : "_listItem_onMousedown",
```

Comment:
Wrong for your second phrase. Just `tr > td` would act the same way as `td` alone - i.e. also trigger on nested tables (of course, if their tags are correctly nested) and that's what I want to avoid/fix.

An ALTERNATIVE to [4b8f6c2](https://github.com/fvgoto/backbone.collectionView/commit/4b8f6c2d14ad4ce577200172d22c0c5d6b9502b8) could also be:
```
"mousedown >li, >tbody>tr>td, >tr>td" : "_listItem_onMousedown",
"dblclick >li, >tbody>tr>td, >tr>td" : "_listItem_onDoubleClick",
```
Thus by explicitly enumerating the `tbody` and with tbody-less variants, separated by coma (`,`). (Side-note: I removed whitespace around `>` for the variants to be more readable. Just double-checked [W3C CSS2](http://www.w3.org/TR/CSS2/selector.html#child-selectors) and learned that these whitespace are optional).

It's up to you if you prefer this verbosity or [4b8f6c2](https://github.com/fvgoto/backbone.collectionView/commit/4b8f6c2d14ad4ce577200172d22c0c5d6b9502b8).